sonic-stormond/scripts/stormond

```diff
@@ -49,6 +50,8 @@ class DaemonStorage(daemon_base.DaemonBase):
         self.log = syslogger.SysLogger(SYSLOG_IDENTIFIER)
         super(DaemonStorage, self).__init__(log_identifier)
 
+        self.log_info("Starting Storage Monitoring Daemon")
```

Comment:
@assrinivasan can you use SysLogger for all logging? https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/syslogger.py#L18

Reply:
Done in latest:

```
root@str2-msn4600c-acs-04:/# /usr/local/bin/stormond
> /usr/local/bin/stormond(53)__init__()
-> self.timeout = STORMOND_PERIODIC_STATEDB_SYNC_SECS
(Pdb) p self.log
<sonic_py_common.syslogger.SysLogger object at 0x7fe318a7f090> <-----------
(Pdb) dir(self.log)
['DEFAULT_LOG_FACILITY', 'DEFAULT_LOG_LEVEL', '__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getstate__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_min_log_level', 'log', 'log_debug', 'log_error', 'log_info', 'log_notice', 'log_warning', 'logger', 'set_min_log_priority']
(Pdb) up
> /usr/local/bin/stormond(387)main()
-> stormon = DaemonStorage(SYSLOG_IDENTIFIER)
(Pdb) n
> /usr/local/bin/stormond(389)main()
-> stormon.log.log_warning("Starting Storage Monitoring Daemon")
(Pdb) p stormon.log
<sonic_py_common.syslogger.SysLogger object at 0x7fe318a7f090> <-----------
(Pdb) quit

```